deploy/us-east-2-dev/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storetheindex-0-storetheindex
+  labels:
+    app: storetheindex
+spec:
+  replicas: 1
+  serviceName: storetheindex
+  selector:
+    matchLabels:
+      app: storetheindex
+      release: storetheindex-0
+  volumeClaimTemplates:
+  template:
+    metadata:
+      labels:
+        app: storetheindex
+        chart: lotus-bundle-0.0.13
+        release: storetheindex-0
+    spec:
+      securityContext:
+        fsGroup: 532
+      volumes:
+        - name: storetheindex-identity-volume
+          secret:
+            secretName: storetheindex-identity
+            defaultMode: 0600
+        - name: storetheindex-config-volume
+          configMap:
+            name: storetheindex-0-storetheindex-config
+        - name: shared-volume
+          persistentVolumeClaim:
+            claimName: storetheindex-pvc
+      initContainers:
+        - args:
+          - -c
+          - envsubst </config/config >/data/storetheindex/config
```

Comment:
I think using `envFrom` and `configmap`of environment variables will eliminate the need for this and the init container alltogher.